hazelcast-sql/pom.xml

```diff
@@ -176,7 +176,9 @@
                                     <include>org.apache.calcite:calcite-linq4j</include>
                                     <include>org.apache.calcite.avatica:avatica-core</include>
                                     <include>org.checkerframework:checker-qual</include>
-                                    <include>com.jayway.jsonpath:json-path</include>
```

Comment:
This is removed, so it should be removed from the relocations section below.

Reply:
Sadly we can't remove it actually, its part of Calcite's core dependencies so it has to be still shaded in.